lib/main_common.pm

```diff
@@ -396,12 +397,7 @@ sub load_boot_tests {
     }
     if ((get_var("UEFI") || is_jeos()) && !check_var("BACKEND", "svirt")) {
         loadtest "installation/data_integrity" if data_integrity_is_applicable;
-        if (is_jeos() && is_x86_64) {
```

Comment:
Is this no longer necessary? If so, can the `jeos/grub2` module be deleted?

Reply:
not yet, there is a dedicated ticket to the cleanup 
* https://progress.opensuse.org/issues/67024